caddy/frankenphp/main.go

```diff
@@ -13,7 +14,8 @@ import (
 )
 
 func init() {
-	maxprocs.Set()
+	//nolint:errcheck
+	maxprocs.Set(maxprocs.Logger(caddy.Log().Sugar().Infof))
```

Comment:
Isn't this more debug information?
```suggestion
	maxprocs.Set(maxprocs.Logger(caddy.Log().Sugar().Debugf))
```